lib/livebook/session.ex

```diff
@@ -68,7 +78,8 @@ defmodule Livebook.Session do
           file: FileSystem.File.t() | nil,
           images_dir: FileSystem.File.t(),
           created_at: DateTime.t(),
-          memory_usage: memory_usage()
+          memory_usage: memory_usage(),
+          secrets: list() | nil
```

Comment:
We can use an empty list instead of nil, no?

Reply:
Yep!